packages/server/src/app.js

```diff
@@ -19,6 +20,8 @@ for (const applyBeforeware of modules.beforewares) {
 
 app.use(cookiesMiddleware());
 
+app.use(i18Init());
```

Comment:
This should be called from within `i18n` module. `i18n` support must be modular. It should not be in the core

Reply:
Fixed.